test/conformance/api/v1/revision_timeout_test.go

```diff
@@ -77,12 +77,14 @@ func TestRevisionTimeout(t *testing.T) {
 	clients := test.Setup(t)
 
 	testCases := []struct {
-		name           string
-		timeoutSeconds int64
-		initialSleep   time.Duration
-		sleep          time.Duration
-		expectedStatus int
-		expectedBody   string
+		name                        string
+		timeoutSeconds              int64
+		responseStartTimeoutSeconds int64
+		idleTimeoutSeconds          int64
```

Comment:
These aren't part of the official Knative spec - so they can't be part of conformance. We'll want these in e2e

Reply:
ok, will move it